rstan/rstan/R/zzz.R

```diff
@@ -18,10 +18,19 @@
 rstan_load_time <- as.POSIXct("1970-01-01 00:00.00 UTC")
 RNG <- 0
 OUT <- 0
+stanc_ctx <- V8::v8()
 
 tbbmalloc_proxyDllInfo <- NULL
 
 .onLoad <- function(libname, pkgname) {
+  assignInMyNamespace("stanc_ctx",  value= V8::v8())
+  stanc_js <- system.file("stanc.js", package = "rstan")
+  if (!file.exists(stanc_js)) {
+    warning(paste0("Default stancjs compiler not found, ",
+                   "downloading the current version from github."))
+    stanc_js <- "https://github.com/stan-dev/stanc3/releases/download/v2.27.0/stanc.js"
```

Comment:
For the reason above, this should use the nightly version of `stanc3`.
```suggestion
    stanc_js <- "https://github.com/stan-dev/stanc3/releases/download/nightly/stanc.js"
```

Reply:
We don't have guarantees the nightly version of stanc will work with rstan. It's very unlikely a user would run into this circumstance but as I said above I'd rather prompt the user to ask if they want us to download the current or nightly version when this doesn't exist. Then we can just place that into where stanc.js normally is

Reply:
As I mentioned above, this fallback is only for testing the nightly version of `stanc3` by removing the local JS. It might be nice to have the option to select a version, but that is unnecessary since only specific version may work. We definitely need an option (or the current fallback) to test the nightly `stanc3` for development.